spec/builder/input_spec.rb

```diff
@@ -72,7 +72,7 @@
         end
 
         it 'should set and "optional" class also when there is presence validator' do
-          @new_post.class.should_receive(:validators_on).with(:title).any_number_of_times.and_return([
+          @new_post.class.stub(:validators_on).with(:title).and_return([
```

Comment:
The difference between `should_receive` and `stub` is that `stub` won't fail the test if it never gets called. I many of these cases, that probably means the stub is not never needed, or that the should_receive is expecting the wrong method.